CAENHVAsynApp/src/channel_parameter.cpp

```diff
@@ -74,6 +74,18 @@ T ChannelParameterBase<T>::getVal() const
     return temp;
 }
 
+template<typename T>
+void ChannelParameterBase<T>::printValOrError(std::ostream& stream) const
+{
+    try {
+        T value = getVal();
+        stream << value;
+    } catch(std::runtime_error& e) {
+        stream << "error unavailable";
```

Comment:
Do you mean "value" rather than "error" unavailable? Is it useful to print   `e.what()` to `stream` ?

Reply:
Ah, I see this is used after a `Value =` hence "value" is implied. Would it still be useful to print the error, or is that duplicating details elsewhere? It may be better to use something like   "<UNAVAILABLE>" or "<ERROR>"

Reply:
I think it's useful to print an obvious message saying error unavailable, but I agree we should also print the error itself so have added the `e.what()` call in